sdk/ai/azure-ai-agents/azure/ai/agents/operations/_patch.py

```diff
@@ -419,28 +419,13 @@ def create(
             raise ValueError("Invalid combination of arguments provided.")
 
         return response
-
+    
     @distributed_trace
-    def create_and_process(
+    def process(
```

Comment:
is process meant to be internal method? if so, we shall rename to _process or _create_and_process_internal or smth like that